test/util/preconditions.js

```diff
@@ -52,7 +52,8 @@ describe('preconditions', function() {
       $.checkArgumentType(1, PrivateKey);
     } catch (e) {
       error = e;
-      e.message.should.equal('Invalid Argument for (unknown name), expected PrivateKey but got number');
```

Comment:
what is `e.message` in other browsers? I'm worried this assertion is failing there